requirements.txt

```
@@ -1,6 +1,6 @@
 # the default package dependencies
 
-numpy>=1.16.6
+numpy>=1.17.5
```

1.17.5 worked -> decreasing

```suggestion
numpy>=1.17.3
```